### ISSUE TITLE
Preset conflict with specific rule

### ISSUE THREAD

**Issue by ikerios** [jsknnr/vrising-server#1]
Hi,
this game follow a kind of  "work as intended" scenario that prioritize the game preset over the specific sub-rule.
So, for example, if GAME_PRESET is set to "StandardPvE" and then a specific rule is set to the file ServerGameSettings.json (like  "TeleportBoundItems": false) the preset rule prevails over it.

To avoid this it should be possible to not pass  -preset and -difficultyPreset in the entrypoint.sh.

My suggestion, to mantain retrocompatibility,  is to avoid to pass the specific parameters only if GAME_PRESET and/or DIFFICULTY are explicitly set to an empty string.

**Comment by jsknnr** [comment]
Yeah I can make that change. Honestly this was just something I threw together quite quick for my own use. Might take me a few days to get to it, but I'll get it done.

**Comment by ambrons** [comment]
Thanks for all the hard work @jsknnr .  Really helped me not have to worry about 99% of the things to get a dedicated server going.

I had a requirement similar to @ikerios.  I have this running for my server without issue.  Only really tested it with k8s.  The work is in #5 .

**Comment by jsknnr** [comment]
Approved and merged. New image is building now.
